src/main/resources/assets/css/marathon.css

```diff
@@ -161,6 +161,10 @@ a,
   margin-right: 10px;
```

Comment:
Let's remove this specific style. I t was something I added for a specific style in a modal at some point, it should not be there anymore. That's my fault.